.repo-tools/Docs_Automation/guids.json

```diff
@@ -0,0 +1,927 @@
+{

```

Comment:
Why not auto-generate this file by scanning through the directories?

Reply:
I would like to, but ultimately the problem is I need to tag each GUID with a version and currently the sample.json does not include that, so this is the method I am currently using. In the future, I plan to make a few more alterations
* If GUID is found in sample.json, but not in GUID, then tool will 
    * request to add it and then request a version to be added.
    * if silent mode, generate an error log showing not found GUID
* Also plan to add running of this script (silent mode) to the weekly Post-production CI run and it will generate a PR of changes

Reply:
So who is responsible for updating this file, and when does it need to be done?  With each new sample added?  If so, we should update the docs on what to do when contributing a new sample as part of this PR.

Reply:
Ultimately the PM or those who they designate. So for now, I will be doing so once a week. I am requesting in the wiki documentation that the guid.json file be updated as part of the initial PR for new samples.